packages/shared/src/utils/benchmark-handler/benchmark-handler.js

```diff
@@ -0,0 +1,16 @@
+export const benchmarkHandler = (id, phase, actual_time, base_duration, start_time, commit_time) => {
```

Comment:
@amir-deriv Are the arguments for this function coming from   `Performance.measure()` that we use in the target actions? a short description for this method would be helpful

Reply:
<img width="622" alt="image" src="https://github.com/binary-com/deriv-app/assets/129206554/2021e87d-9bdd-4531-a42c-f89c83773813">

Now it is showing the description of the function @shayan-deriv

Reply:
so what about this doc? I'm confused :smiling_face_with_tear: 
https://wikijs.deriv.cloud/en/Frontend/deriv-app/measure-performance-of-an-action

Reply:
@amir-deriv now I get it. thanks for the explanation :pray: